### PR TITLE
Modified to use STRV_MAKE

### PR DESCRIPTION
**Pull request by taniishkaaa**
Follow up for #27100

### CODE REVIEW

File: src/test/test-env-util.c [hunk 1]
```diff
@@ -452,4 +452,10 @@ TEST(getenv_steal_erase) {
         assert_se(r > 0);
 }
 
+TEST(strv_env_name_is_valid) {
+        assert_se(strv_env_name_is_valid(STRV_MAKE("HOME", "USER", "SHELL", "PATH", NULL)));
+        assert_se(!strv_env_name_is_valid(STRV_MAKE("", "PATH", "home", "user", "SHELL", NULL)));
+        assert_se(!strv_env_name_is_valid(STRV_MAKE("HOME", "USER", "SHELL", "USER", NULL)));
```
Comment (daandemeyer) [inline src/test/test-env-util.c]: The trailing NULLs can be dropped, they're added automatically by `STRV_MAKE()`

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]

**Comment by bluca** [comment]
also needs a rebase

**Comment by taniishkaaa** [comment]
Closing this PR in favor of #27130

**Comment by bluca** [comment]
> Closing this PR in favor of #27130

For future reference, please don't close PRs and reopen, as the context is lost. It is not needed, you can simply force push to the same branch.

**Pull request closed without merging**